src/NumericalAlgorithms/LinearOperators/PowerMonitors.cpp

```diff
@@ -27,21 +28,21 @@ void power_monitors(const gsl::not_null<std::array<DataVector, Dim>*> result,
   double slice_sum = 0.0;
   size_t n_slice = 0;
   size_t n_stripe = 0;
-  for (size_t sliced_dim = 0; sliced_dim < Dim; ++sliced_dim) {
-    n_slice = mesh.extents().slice_away(sliced_dim).product();
-    n_stripe = mesh.extents(sliced_dim);
+  for (size_t d = 0; d < Dim; ++d) {
```

Comment:
Why are you renaming these? I find `sliced_dim` to be a fairly clear variable name, same with `index`

Reply:
my fault, I said `i` and `d` are fine for loop variables